### PR TITLE
enh(plugin): VTOM REST API metricsv2

### PR DESCRIPTION
**Pull request by itoussies**
Updating the plugin with the new metric naming format.

### CODE REVIEW

**Comment by itoussies** [comment]
Oups I forgot something. Fixing it right now.

**Comment by itoussies** [comment]
Ok now it should be good 😅

File: apps/vtom/restapi/mode/jobstatus.pm [hunk 1]
```diff
@@ -83,72 +95,71 @@ sub set_counters {
...
                 closure_custom_perfdata => sub { return 0; },
-                closure_custom_threshold_check => \&catalog_status_threshold,
+                closure_custom_threshold_check => \&catalog_status_threshold_ng,
             }
         },
     ];
     
     $self->{maps_counters}->{global} = [
-        { label => 'total-error', set => {
+        { label => 'total-error', nlabel => 'error.total.count', set => {
```
Comment (garnier-quentin) [inline apps/vtom/restapi/mode/jobstatus.pm:123]: could prefix ? jobs.errors.count ?

File: apps/vtom/restapi/mode/jobstatus.pm [hunk 2]
```diff
@@ -83,72 +95,71 @@ sub set_counters {
...
                 perfdatas => [
-                    { label => 'total_error', value => 'error', template => '%s',
-                      min => 0, max => 'total' },
-                ],
+                    { label => 'total_error', template => '%s', min => 0, max => 'total' }
+                ]
             }
         },
-        { label => 'total-running', set => {
+        { label => 'total-running', nlabel => 'running.total.count', set => {
```
Comment (garnier-quentin) [inline apps/vtom/restapi/mode/jobstatus.pm:131]: jobs.running.count

File: apps/vtom/restapi/mode/jobstatus.pm [hunk 3]
```diff
@@ -83,72 +95,71 @@ sub set_counters {
...
                 perfdatas => [
-                    { label => 'total_running', value => 'running', template => '%s',
-                      min => 0, max => 'total' },
-                ],
+                    { label => 'total_running', template => '%s', min => 0, max => 'total' }
+                ]
             }
         },
-        { label => 'total-unplanned', set => {
+        { label => 'total-unplanned',  nlabel => 'unplanned.total.count', set => {
```
Comment (garnier-quentin) [inline apps/vtom/restapi/mode/jobstatus.pm:139]: jobs.unplanned.count

File: apps/vtom/restapi/mode/jobstatus.pm [hunk 4]
```diff
@@ -83,72 +95,71 @@ sub set_counters {
...
                 perfdatas => [
-                    { label => 'total_unplanned', value => 'unplanned', template => '%s',
-                      min => 0, max => 'total' },
-                ],
+                    { label => 'total_unplanned', template => '%s', min => 0, max => 'total' }
+                ]
             }
         },
-        { label => 'total-finished', set => {
+        { label => 'total-finished', nlabel => 'finished.total.count', set => {
```
Comment (garnier-quentin) [inline apps/vtom/restapi/mode/jobstatus.pm:147]: jobs.finished.count

File: apps/vtom/restapi/mode/jobstatus.pm [hunk 5]
```diff
@@ -83,72 +95,71 @@ sub set_counters {
...
                 perfdatas => [
-                    { label => 'total_finished', value => 'finished', template => '%s',
-                      min => 0, max => 'total' },
-                ],
+                    { label => 'total_finished', template => '%s', min => 0, max => 'total' }
+                ]
             }
         },
-        { label => 'total-coming', set => {
+        { label => 'total-coming', nlabel => 'coming.total.count', set => {
```
Comment (garnier-quentin) [inline apps/vtom/restapi/mode/jobstatus.pm:155]: jobs.coming.count

**Review by garnier-quentin — APPROVED** [review]

**Pull request merged**